jenkins/github_helpers.py

```diff
@@ -215,6 +217,9 @@ def create_pull_request(self, repository, title, body, base, head, user_reviewer
                 "https://github.com/{}/pull/{}".format(repository.full_name, pull_request.number)
             ) from e
 
+        if pull_request.title == 'Python Requirements Update' and repository.name in ['repo-health-data']:
```

Comment:
Add a comment in the code that we are doing discovery work, so limiting this to `repo-health-data`